dojo/tools/trivy_operator/vulnerability_handler.py

```diff
@@ -83,8 +83,9 @@ def handle_vulns(self, labels, vulnerabilities, test):
                 dynamic_finding=False,
                 service=service,
                 file_path=file_path,
-                tags=finding_tags,
             )
```

Comment:
Since `finding_tags` is defined as a list on line 35, it'll never be `""`. You could do something like this instead:

```python
tags=[tag for tag in finding_tags if tag != ""]
```